code/style.css

```diff
@@ -10,49 +46,657 @@ p {
 }
 
 .cards {
-  margin: 50px auto;
-  width: 90%;
+  margin: 65px auto;
+  width: 100%;
   height: 600px;
-  background: #ccc;
   display: flex;
-  justify-content: space-around;
+  justify-content: space-evenly;
   align-items: center;
+  position: relative;
 }
 
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background: #1a1110;
+  opacity: 0.9;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
 }
 
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: #545151;
+  opacity: 0.9;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background: #62434c;
+  opacity: 0.9;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
 }
 
-button {
-  padding: 8px 16px;
-  background: crimson;
-  border: none;
+#crimeButton {
```

Comment:
I suggest you use only classes and not id’s to be consistent when styling. I recall that Mathilda or Maks once gave me this feedback. 🤓